app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/common/OverlayView.java

```diff
@@ -49,6 +47,9 @@ public OverlayView(Context context, AttributeSet attrs, int defStyle) {
         Point displaySize = new Point();
         display.getSize(displaySize);
         maxScrnWidth = displaySize.x;
+
+        // stop propagation
+        setOnClickListener(v -> {});
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen-android/blob/e94f55687874304ef24ba5dd31aeac298e600a88/app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/common/OverlayView.java#L52"></span>’}’ is not preceded with whitespace.</td>
    </tr>
  </tbody>
</table>